src/main/java/io/cdap/plugin/gcp/bigquery/util/BigQueryUtil.java

```diff
@@ -512,10 +518,6 @@ public static Map<String, String> configToMap(Configuration config) {
   public static ValidationFailure validateArraySchema(Schema arraySchema, String name, FailureCollector collector) {
     Schema nonNullableSchema = arraySchema.isNullable() ? arraySchema.getNonNullable() : arraySchema;
     Schema componentSchema = nonNullableSchema.getComponentSchema();
-    if (componentSchema.isNullable()) {
```

Comment:
I think this is still needed ? we are handling null array instead of array with null element ?